src/shared/ui/Splash.tsx

```diff
@@ -16,6 +24,8 @@ const SplashScreen = ({ isAppReady }: { isAppReady: boolean }) => {
   const containerOpacity = useRef(new Animated.Value(1)).current;
   const imageOpacity = useRef(new Animated.Value(0)).current;
 
+  const appVersion = DeviceInfo.getVersion();
```

Comment:
Please use this line instead:
`const appVersion = ENV ? `${APP_VERSION} ${ENV}` : APP_VERSION,`

from:
<img width="461" alt="image" src="https://github.com/ChildMindInstitute/mindlogger-app-refactor/assets/20073193/d8888799-4248-40b9-92ee-b90207b050ec">